study-group/Toast-Implementation-Studies.md

```diff
@@ -407,10 +520,105 @@ The library allows showing multiple toasts, and gives the developer specific con
 - Comes with default icons, can change by modifying CSS.
 - **Note**: On mobile there's currently no simple way to stop the timeout, accomplished by hover on desktop.
 
-## Bootstrap
+## [Bootstrap Toast](https://getbootstrap.com/docs/4.2/components/toasts/)
```

Comment:
Somehow the bootstrap study is in here too. Rebase? Or is it me not knowing how to use github?

Reply:
Hmm you're right, I'm not really sure how I got that wrong. How would you recommend I solve this (my Git-Fu is somewhat lacking at the moment)?

Reply:
I'm awake now, and can help in person :)

Reply:
Got it rebased, thanks!